package.json

```diff
@@ -7,13 +7,13 @@
   },
   "license": "Apache-2.0",
   "scripts": {
-    "start": "concurrently \"npm run start:backend\" \"npm run start:frontend\"",
+    "start": "concurrently \"npm run start:backend\" \"npm run start:frontend --kubernetes-dashboard:bind_address=$npm_package_config_bind_address\"",
     "start:https": "concurrently \"npm run start:backend:https\" \"npm run start:frontend:https\"",
-    "start:frontend": "npm run postversion && ng serve --aot --progress=false --proxy-config aio/proxy.conf.json --host=0.0.0.0",
-    "start:frontend:https": "node aio/scripts/version.js && ng serve --progress=false --aot --proxy-config aio/https-proxy.conf.json --ssl --host=0.0.0.0",
+    "start:frontend": "echo $npm_package_config_bind_address && npm run postversion && ng serve --aot --progress=false --proxy-config aio/proxy.conf.json --host=$npm_package_config_bind_address",
```

Comment:
Is this `echo` a debug leftover?

Reply:
Yes, it's my mistake. Sorry. I'll remove it soon.